blocks/trending-news/trending-news.js

```diff
@@ -0,0 +1,113 @@
+import { getTrendingNews } from '../../scripts/mockapi.js';
+
+export default function decorate(block) {
+  const newsData = getTrendingNews();
+  block.textContent = '';
+
+  function createNewsItem(news) {
```

Comment:
for better readability, we can place this function outside the decorate method.

Reply:
done. moved